src/commands/createFunctionApp/stacks/FunctionAppStackStep.ts

```diff
@@ -26,27 +28,33 @@ export class FunctionAppStackStep extends AzureWizardPromptStep<IFunctionAppWiza
                     parameterDisplayDescription: 'The runtime stack to use for the function app.'
                 }
             };
-            const picks = this.getPicks(context);
+
+            const picks = isFlex ? flexStacksQuickPicks : await this.getPicks(context);
```

Comment:
I'm running into some type/compile errors when I run here.

Reply:
I stupidly put the return type as `{result: string}` rather than `{runtime: string}`. That should go away now; also moved the stacks out of the constants since it's only used here and is temporary anyway.